### PR TITLE
Travis and Gemfile - Bundler version hinting

### PR DESCRIPTION
**Pull request by olleolleolle**
This PR contains Travis build fixes from #47 about Gemfiles and Bundler version hinting.

### CODE REVIEW

**Comment by olleolleolle** [comment]
@mikker This is the smallest change that would allow us to build on the existing platforms, I believe.

**Comment by olleolleolle** [comment]
The current matrix build errors are not about "that gem can't install on this platform".

File: gemfiles/Gemfile.base [hunk 1]
```diff
@@ -15,12 +17,13 @@ gem 'simplecov'
 
 gem 'redis'
 gem 'fakeredis'
-gem 'sqlite3'
+gem 'sqlite3', platform: :mri
```
Comment (olleolleolle) [inline gemfiles/Gemfile.base]: Note: This is not strictly necessary in this change. I could get rid of it.

**Comment by mikker** [comment]
lgtm 💙💚💛💜❤️

**Pull request merged**